client/components/UserSettings/AdminSettings.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import { LoadingStatus, useTriggerLoad } from '../common/LoadingStatus';
+import { useThemedModal, THEMES } from '../../hooks/useThemedModal';
+import { dates } from 'shared';
+
+const AdminSettings = ({ latestTestPlanVersion, refetch }) => {
+  const { triggerLoad, loadingMessage } = useTriggerLoad();
+  const {
+    themedModal,
+    showThemedModal,
+    setShowThemedModal,
+    setThemedModalTitle,
+    setThemedModalContent,
+    setThemedModalType
+  } = useThemedModal({
+    type: THEMES.SUCCESS,
+    title: 'Success!'
+  });
+
+  const handleImportTests = async () => {
+    await triggerLoad(async () => {
+      try {
+        const response = await fetch('/api/test/import', { method: 'POST' });
```

Comment:
I know that it would probably be a relatively significant lift to get a progress bar on this  but it is difficult to see the infinite spinner with such a long running operation. My system took a few minutes to complete the task and I was concerned that something had broken. If the progress bar would be too much work then perhaps we could add a note that "this could take a few minutes"?

Reply:
Seems fair!

Reply:
Opted to add a note instead in https://github.com/w3c/aria-at-app/pull/1290/commits/a9dae3ab42039199b8be4906613f43df803f4012. Let me know if this feels better to you

Reply:
Yep this feels much better. Thanks!